eslint.config.js

```diff
@@ -0,0 +1,39 @@
+// eslint.config.js
+const { defineConfig } = require('eslint-define-config');
+
+// ESLint recommended rules directly defined
+const recommendedRules = {
+    'no-unused-vars': ['error', { argsIgnorePattern: '^(err|req|res|next)$' }],
+    'no-mixed-requires': ['error', { allowCall: true }],
+    quotes: ['error', 'single'],
+    'no-trailing-spaces': 'error',
+    indent: 'error',
+    'linebreak-style': ['error', 'unix'],
+    semi: ['error', 'always'],
+    'brace-style': ['error', '1tbs', { allowSingleLine: true }],
+    'keyword-spacing': 'error',
+    'space-before-blocks': 'error',
+    'space-before-function-paren': ['error', { anonymous: 'always', named: 'never' }],
+    'no-mixed-spaces-and-tabs': 'error',
+    'comma-spacing': ['error', { before: false, after: true }],
+    'key-spacing': ['error', { beforeColon: false, afterColon: true }],
+    'one-var': ['error', { initialized: 'never' }],
+    'no-var': 'error',
+};
+
+// Define the ESLint configuration
+module.exports = defineConfig([
+    {
+        languageOptions: {
+            globals: {
+                module: 'readonly',
```

Comment:
We can use the package `globals` as a `devDependency` to set up the Node globals same as what our internal ESLint config does instead of needing to define any individually

Reply:
This hasn't been resolved